Source/ACE/Managers/LandblockManager.cs

```diff
@@ -86,7 +85,9 @@ private static Landblock GetLandblock(LandblockId landblockId, bool propogate)
                     if (landblocks[x, y] == null)
                     {
                         // load up this landblock
-                        Landblock block = new Landblock(landblockId);
+                        Landblock block = new Landblock(landblockId, x, y);

```

Comment:
How do I get x, and y from landblock ?
Keep in mind I am trying to draw this kn a grid of 256x256 in pixels